keyboards/silverbullet44/keymaps/FKML/keymap.c

```diff
@@ -0,0 +1,148 @@
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |Shift |   A  |   S  |   D  |   F  |   G  |                                  |   H  |   J  |   K  |   L  |   ;  |  '   |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * | CMD  |   Z  |   X  |   C  |   V  |   B  |-------.-------.  ,---------------|   N  |   M  |   ,  |   .  |   /  |  -   |
+ * `-----------------------------------------/       /       /   \       \      \-----------------------------------------'
+ *                            | CAPS  | LOWER| Space/  Bcsp /     \    .  \ Ent | RAISE|LEDOff|
+ *                            `--------------- -------------'       '--------------------------'
+ */
+ [_QWERTY] = LAYOUT( 
+     KC_TAB,   KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,                      KC_Y,   KC_U,    KC_I,    KC_O,   KC_P,    KC_EQL,  
+    KC_LSFT,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,                      KC_H,   KC_J,    KC_K,    KC_L,   KC_SCLN, KC_QUOT, 
+    KC_LCMD,   KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,                      KC_N,   KC_M,    KC_COMM, KC_DOT, KC_SLSH, KC_MINS, 
+                            KC_CLCK,  LOWER, KC_SPC, KC_BSPC,    KC_DOT,  KC_ENT,  RAISE, RGB_TOG 
+  ),
+
+/* LOWER
+ * ,-----------------------------------------.                                  ,-----------------------------------------.
+ * |   `  |   !  |   @  |   #  |   $  |   %  |                                  |   ^  |   &  |   *  |   (  |   )  |   |  |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |                                  |   1  |   2  |   3  |   4  |   5  |      |
+ * |------+------+------+------+------+------+                                  +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |-------.-------.  ,---------------|   6  |   7  |   8  |   9  |   0  |      |
+ * `-----------------------------------------/       /       /   \       \      \-----------------------------------------'
+ *                             |      |      |      /       /     \       \     |      |      |
+ *                             `----------------------------'      '--------------------------'
+ */
+ [_LOWER] = LAYOUT( 
+  KC_GRV , KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC,                  KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LPRN, KC_RPRN, KC_PIPE, 
+  _______, _______, _______, _______, _______, _______,                     KC_1,     KC_2,     KC_3,     KC_4,    KC_5, _______, 
+  _______, _______, _______, _______, _______, _______,                     KC_6,     KC_7,     KC_8,     KC_9,    KC_0, _______, 
+                             _______, _______, _______, ADJUST,  _______, _______, _______, _______ 
```

Comment:
ADJUSTを使わないようなら消してくれると…！
```suggestion
                             _______, _______, _______, _______,  _______, _______, _______, _______ 
```